### ISSUE TITLE
Not able to replicate the 80% top-1 accuracy on CUB2011 for SWSL ResNeXt-50 32x4d model

### ISSUE THREAD

**Issue by ZhengRui** [facebookresearch/semi-supervised-ImageNet1K-models#2]
I tried to replicated the transfer learning experiments on CUB2011 as presented on 
https://ai.facebook.com/blog/billion-scale-semi-supervised-learning/ , it mentioned by fine-tuning the semi weakly supervised resnet50 pre-trained model, it can reach 80% top-1 accuracy.

First, I experimented with densenet201, which has similar performance as resnet50 on imagenet (around 77% top-1 accuracy). By freezing all the layers and only fine-tune the last classification layer on CUB2011, i am able to get 73.5% top-1 accuracy, which makes sense compared to the 73.3% top-1 accuracy for resnet50. (start from learning rate 0.001, decreased learning rate at epoch 30 to 0.0002 and reach 73.47% at epoch 33, with batch size 1072 using checkpoint_sequential)

Second, I tried to achieve 80% top-1 accuracy by using the semi-weakly supervised learning resnet50 pre-trained models, from this repo i chose `semi_weakly_supervised_resnext50_32x4-72679e44.pth` and did the same experiment as above, but I could only get 72% top-1 accuracy. I think as pytorch vision document mentioned, all pre-trained pytorch models have same preprocessing steps, so I didn't change the preprocessing step. (start from learning rate 0.005, decrease at epoch 22 to 0.001 and reach around 72% after few epochs but i feel it is already saturized and there is a big gap to 80%. Batch size is 640 because I haven't enable checkpoint_sequential on this model, but i don't think this will be the deciding factor for small dataset as CUB2011, and I repeated the experiment with different learning rates and got similar ~72% performance). I am wondering how to be able to replicate the 80% top-1 accuracy, I appreciate if you can shed some light on this replication, thanks!

**Comment by fbzekiyalniz** [comment]
Hi Zheng, 

First of all, I should note that you are using ResNext-50-32x4 model for your experiments, not ResNet-50. 

Second, we only released the ResNet and ResNext models after fine-tuning  on the ImageNet training dataset. We have not released the pre-trained models. In the CUB experiments, we use those pre-trained models directly for fine-tuning on CUB2011 dataset. Therefore I am not sure whether you can get the same numbers by fine-tuning the ImageNet fine-tuned semi/semi-weakly supervised models. 

Once you dowmload the correct model, please try the following training parameters below and let me know if you are able to get comparable results. 

  Batch size per GPU: 24
  Number of training epochs: 900      
  Epoch size: 5994   
  Base learning rate per GPU: 0.00234375
  Weight decay: 0.001
  Learning rate decay rate: 0.1 (i.e., reduce learning rate 10x after each decay step below)
  Learning rate decay steps: [300, 300, 200, 200]   (no warm up epochs)

**Comment by ZhengRui** [comment]
Hi, thank you so much for the clarification, I guess as you mentioned the reason is I used the ImageNet fine-tuned model instead of the model before fine-tuning on ImageNet. Are you planning to release the pre-trained models that are before fine-tuning on ImageNet?

**Comment by fbzekiyalniz** [comment]
Unfortunately there is no such plan on releasing the pre-trained models. I will definitely let you know if plans change.
